### PR TITLE
 advisory tiles

### PR DESCRIPTION
**Pull request by fadeomar**
#57 
## create a component for advisory Tiles 
contain a group of single advisory component 

for desktop : 
4 tiles  by row as squares  with Approximately ~ 250 px width 
background white without border   
for mobile : 
single tile take all the width 

### to view the component :+1: 
  * import it in the index-page template 
```
import AdvisoryTiles from '../components/AdvisotyTiles';
```

### CODE REVIEW

File: src/components/AdvisotyTiles/advisoryTile.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+const AdvisoryTile = ({ item }) => {
+  const classes = useStyles();
+
+  return (
+    <Link to={`/${item.title}`} className={classes.link}>
+      <Card className={classes.root} elevation={0}>
+        <CardMedia className={classes.media} src={item.Logo} component="img" title={item.title} />
+        <CardContent>
+          <Typography variant="subtitle1" component="h2" align="center" color="blcak">
```
Comment (JoseChungFOS) [inline src/components/AdvisotyTiles/advisoryTile.js]:
```suggestion
          <Typography variant="subtitle1" component="h2" align="center" color="black">
```

File: src/components/AdvisotyTiles/advisoryTile.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      boxShadow:
+        ' 0px 3px 1px -2px rgba(0,0,0,0.2), 0px 2px 2px 0px rgba(0,0,0,0.14), 0px 1px 5px 0px rgba(0,0,0,0.12)',
+      transform: 'translateY(-2px)',
+    },
+    '@media (max-width: 1024px)': {
+      maxWidth: '100vw',
+      width: '85vw',
+      display: 'flex',
+      flexDirection: 'row',
+      justifyContnet: 'center',
```
Comment (JoseChungFOS) [inline src/components/AdvisotyTiles/advisoryTile.js]:
```suggestion
      justifyContent: 'center',
```

File: src/components/AdvisotyTiles/advisoryTile.js (hunk 1 above)
Comment (fadeomar) [reply src/components/AdvisotyTiles/advisoryTile.js]: embarrassing :)

File: src/components/AdvisotyTiles/advisoryTile.js (hunk 2 above)
Comment (fadeomar) [reply src/components/AdvisotyTiles/advisoryTile.js]: :)

**Pull request merged**